### ISSUE TITLE
Please port usb2uartjtag from RV-Debugger-BL702 to BL616

### ISSUE THREAD

**Issue by edanuff** [sipeed/M0S_BL616_example#11]
Would be great to be able to use the usb2uartjtag code from the RV Debugger with this but it doesn't compile with the new buoffalo sdk.

**Comment by Chandler-Kluser** [comment]
I would appreciated so much, too

I have lost my BL616 stock firmware for my tang nano 20k, and it would help me A LOT!!

**Comment by gamelaster** [comment]
Sipeed doesn't maintain those products anymore (as far as I know)

**Comment by Chandler-Kluser** [comment]
well, this doesn't mean they can share firmware or even the source code. I am not asking them to maintain any product on the shelf, btw

**Comment by gamelaster** [comment]
@Chandler-Kluser that's true, although, for this firmware, SiPeed mostly shared their firmwares and sources, but for some specific stuff, they stopped... Not sure why.

**Comment by Chandler-Kluser** [comment]
that`s said, thanks for the information!!
